app/controllers/v0/institutions_controller.rb

```diff
@@ -29,7 +29,7 @@ def index
     # GET /v0/institutions/20005123
     def show
       resource = Institution.version(@version[:number]).find_by(facility_code: params[:id])
-
+      resource[:ope] = resource[:ope].tr('\"', '') if (resource.present? && resource[:ope].present?)
```

Comment:
Is this a candidate for inclusion in the model or possibly something that can be worked into a converter process?

Reply:
@jkassemi We cannot write this piece of code in to converter process,  as in CSV we need quotes for OPE_ID and coming to model I think it does not make big difference as we are querying the data here.

Reply:
What about the serializer?

Would like, where possible, to maintain focus in the controller on authentication, input sanitization, data lookup, and orchestrating rendering. This seems like presentation logic within the controller.

Reply:
Thanks, Updated the code